.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "FuelLabs/fuel-ts" }
+  ],
+  "commit": false,
+  "fixed": [["@fuel-wallet/*"]],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "master",
+  "updateInternalDependencies": "patch",
+  "ignore": ["app", "page"],
```

Comment:
Should we ignore, store, config, and fuelhat? The idea is to release just `sdk` and `types` for now.

Reply:
✅